test/libyul/yulOptimizerTests/fullSuite/devcon_example.yul

```diff
@@ -21,8 +21,9 @@
 //     {
 //         let _1 := calldataload(0)
 //         let sum := 0
+//         let length := calldataload(_1)
```

Comment:
Why is `_1` not rematerialized here? Is it because it is an SSA variable?

Reply:
Because _1 is used twice